scripts/benchmark.js

```diff
@@ -14,126 +14,181 @@ const fs     = require("fs");
 const path   = require("path");
 const compile = require("google-closure-compiler-js").compile;
 
-const filename = process.argv[2];
-if (!filename) {
-  console.error("Error: No filename specified");
-  process.exit(1);
+const packagename = process.argv[2];
+var filename = process.argv[3];
+
+if (!packagename) {
+	console.error("Error: No package specified");
+	process.exit(1);
 }
 
+console.log("package: " + packagename);
+
+const pathToScripts = "./scripts";
+
 const table = new Table({
-  head: ["", "raw", "raw win", "gzip", "gzip win", "parse time", "run"],
-  chars: {
-    top: "",
-    "top-mid": "" ,
-    "top-left": "" ,
-    "top-right": "",
-    bottom: "" ,
-    "bottom-mid": "" ,
-    "bottom-left": "" ,
-    "bottom-right": "",
-    left: "",
-    "left-mid": "",
-    mid: "",
-    "mid-mid": "",
-    right: "" ,
-    "right-mid": "",
-    middle: " ",
-  },
-  style: {
-    "padding-left": 0,
-    "padding-right": 0,
-    head: ["bold"],
-  },
+	head: ["", "raw", "raw win", "gzip", "gzip win", "parse time", "run"],
+	chars: {
+		top: "",
+		"top-mid": "" ,
+		"top-left": "" ,
+		"top-right": "",
+		bottom: "" ,
+		"bottom-mid": "" ,
+		"bottom-left": "" ,
+		"bottom-right": "",
+		left: "",
+		"left-mid": "",
+		mid: "",
+		"mid-mid": "",
+		right: "" ,
+		"right-mid": "",
+		middle: " ",
+	},
+	style: {
+		"padding-left": 0,
+		"padding-right": 0,
+		head: ["bold"],
+	},
 });
 
 let results = [];
 
-const code = fs.readFileSync(filename, "utf8");
-const gzippedCode = zlib.gzipSync(code);
-
-function test(name, callback) {
-  console.log("testing", name);
-
-  const start = Date.now();
-  const result = callback(code);
-  const end = Date.now();
-  const run = end - start;
-
-  fs.writeFileSync(".test_gen_" + name + ".js", result);
+var exec = child.exec;
+var code, gzippedCode;
+
+function installPackage() {
```

Comment:
I'm wondering whether we should offer some kind of offline switch so that if you're repeated running the benchmark it doesn't keep installing/uninstalling from npm?